src/main/scala/interfaces/Afe.scala

```diff
@@ -63,6 +63,13 @@ class SidebandAfeIo(
 
   /** Enable sideband receivers. */
   val rxEn = Output(Bool())
+
+  /** Sideband PLL Lock.
+   *
+   * Indicates whether the sideband clock is stable.
+   *
+   */
+  val sbPllLock = Output(Bool())
```

Comment:
can just call this `pllLock`, since the "sideband" part is implied by this signal being part of `SidebandAfeIo`